src/Tests/Microsoft.NET.Build.Tests/KnownRuntimePackTests.cs

```diff
@@ -106,6 +106,40 @@ public void RuntimePackWithLabelIsSelected()
 
         }
 
+        [Fact]
+        public void AspNetRuntimePackIsNotRestoredForAndroid()
+        {
+            var testProject = new TestProject()
+            {
+                TargetFrameworks = "net6.0",
```

Comment:
nit: don't we have a current framework field now to make it easier to update test TFMs?